scripts/xtensa-build-zephyr.py

```diff
@@ -15,7 +15,8 @@
 import os
 import warnings
 # anytree module is defined in Zephyr build requirements
-from anytree import Node, RenderTree
+from anytree import AnyNode, RenderTree
+import re
```

Comment:
Yes, we can drop `re`

Reply:
@aborisovich @marc-hb For this AnyNode bug fix
It's because Node can only have `name` and `parent`. When using `name' to store folder name and search for parent , `sof_jsl.ri` and `sof-jsl.ri` find the wrong parent, and show the wrong laytout like this:
```
build-sof-staging
├── sof
│   ├── apl
│   │   ├── community
│   │   │   ├── sof-apl.ri
│   │   │   ├── sof-tgl.ri
│   │   │   └── sof-jsl.ri
│   │   └── sof-apl.ldc
│   ├── tgl
│   │   ├── community
│   │   └── sof-tgl.ldc
│   └── jsl
│       ├── sof-jsl.ldc
│       └── community
└── tools
    ├── sof-logger
    └── cavstool.py
```

But AnyNode can allow us to define both `name` and `short_name`. We use `name` for accurate path match to find the correct parent, and use 'short_name` to show the layout. The correct layout will be: 
```
build-sof-staging
├── sof
│   ├── apl
│   │   ├── community
│   │   │   └── sof-apl.ri
│   │   └── sof-apl.ldc
│   ├── tgl
│   │   ├── community
│   │   │   └── sof-tgl.ri
│   │   └── sof-tgl.ldc
│   └── jsl
│       ├── sof-jsl.ldc
│       └── community
│           └── sof-jsl.ri
└── tools
    ├── sof-logger
    └── cavstool.py
```